gn3/api/correlation.py

```diff
@@ -1,18 +1,36 @@
 """Endpoints for running correlations"""
-from unittest import mock
-
 from flask import jsonify
 from flask import Blueprint
 from flask import request
 
 from gn3.computations.correlations import compute_all_sample_correlation
 from gn3.computations.correlations import compute_all_lit_correlation
 from gn3.computations.correlations import compute_all_tissue_correlation
-
+from gn3.computations.correlations import map_shared_keys_to_values
+from gn3.db_utils import database_connector
```

Comment:
Hmmm... IMHO you don't need the db_utils. All it does it give back a conn right. Remember that the db params are set in a settings file right? Creating that conn object is a one liner, which I don't think needs a module of it's own. Problem with this approach is that it can lead to bloat; Makes sense?